elements/urth-core-function/urth-core-function.html

```diff
@@ -434,15 +440,6 @@
                 }
             },
 
-            _tryInvoke: function() {
```

Comment:
The idea behind `tryInvoke` was that it would not cause the `Error` that is seen when calling `invoke`. The `Error` thrown by `invoke` was by design in case there is a programatic call to `invoke` when it is not ready. We do not want that error when `urth-core-function` is invoking itself.